PGM/src/main/java/tc/oc/pgm/shop/strategy/PlayerPaymentStrategy.java

```diff
@@ -0,0 +1,49 @@
+package tc.oc.pgm.shop.strategy;
+
+import tc.oc.pgm.filters.Filter;
+import tc.oc.pgm.match.Competitor;
+import tc.oc.pgm.match.MatchPlayer;
+import tc.oc.pgm.shop.purchasable.Purchasable;
+
+/**
+ * Strategy which only allows the initiating {@link MatchPlayer} to contribute.
+ */
+public class PlayerPaymentStrategy extends PaymentStrategyImpl<MatchPlayer> {
+    final MatchPlayer player;
+    /**
+     * If this strategy can be carried with the player when they switch {@link tc.oc.pgm.match.Party Parties}.
+     */
+    final boolean crossTeam;
+    /**
+     * If this strategy is persistent across player logins (during the same match).
+     */
+    final boolean persistent;
+
+    // Data about player when the strategy was (re)started.
+    final Competitor competitor;
```

Comment:
When a player changes an if statement below checks if the config is persistent and if so disregards it anyway.

I do need to clear all non-persisted strategies on change to prevent conflicts, though.